src/main/java/com/paskar/email/application/EmailDeliveryApplication.java

```diff
@@ -0,0 +1,13 @@
+package com.paskar.email.application;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.scheduling.annotation.EnableScheduling;
+
+@SpringBootApplication
```

Comment:
FYI: org.springframework.boot.autoconfigure.SpringBootApplication#proxyBeanMethods
nice practice to use "light beans" where possible.